### PR TITLE
Booking tweaks 1

### PR DESCRIPTION
**Pull request by Demotive**
Initial round of visual tweaks to booking.
- Redo 'booking panel' to make it more compact and have an obvs call to action :|
- Redo the appointment options layout a bit

screen shot 2015-11-03 at 12 35 32

### CODE REVIEW

File: app/views/book-an-appointment/confirm-appointment.html [hunk 1]
```diff
@@ -17,6 +17,7 @@ <h1 class="heading-xlarge">
     ui_element_macros.appointment_summary({
       appointment_date: appointment.appointment_date,
       appointment_time: appointment.appointment_time,
+      appointment_days_away: appointments.appointment.appointment_days_away,
```
Comment (evilstreak) [inline app/views/book-an-appointment/confirm-appointment.html]: This isn't the right local, `appointment`

File: app/views/book-an-appointment/appointment-confirmed.html [hunk 2]
```diff
@@ -17,6 +17,7 @@ <h1 class="heading-xlarge">
     ui_element_macros.appointment_summary({
       appointment_date: appointment.appointment_date,
       appointment_time: appointment.appointment_time,
+      appointment_days_away: appointments.appointment.appointment_days_away,
```
Comment (evilstreak) [inline app/views/book-an-appointment/appointment-confirmed.html]: This isn't the right local, `appointment`

File: app/views/includes/ui_element_macros.html [hunk 3]
```diff
@@ -1,5 +1,55 @@
...
+        </div>
+      </div>
+      <div class="cta">
+        <a href="{{ i.link_url }}" class="button button-secondary-cta">Book this appointment</a>
+      </div>
+    </div>
+
+  </div>
+
+  <!--
```
Comment (evilstreak) [inline app/views/includes/ui_element_macros.html]: What's the reason for commenting this out rather than just deleting it?

Comment (Demotive) [inline app/views/includes/ui_element_macros.html]: Fear of failure. But having failed elsewhere...

**Comment by evilstreak** [comment]
Broadly :+1:, a few variable names need fixing and a couple of appointment blocks don't have the new `days_away` value.

**Pull request merged**